lua/entities/gmod_wire_expression2/core/player.lua

```diff
@@ -958,6 +985,70 @@ E2Lib.registerEvent("playerChangedTeam", {
 	{ "NewTeam", "n" }
 })
 
+--[[--------------------------------------------------------------------------------------------]]--
+
+__e2setcost(1)
+
+--- Returns <this>'s mouse delta on the x-axis.
```

Comment:
You can remove all of the comments above the `e2function`s, since e2 docs won't use this and devs can just look up the glua function themselves.

Reply:
I have now removed all of the comments above the e2function's and added descriptions to e2descriptions.lua.